src/provider/store/Store.ts

```diff
@@ -22,11 +23,17 @@ export class Store {
     public readonly onAction: Signal<[RootAction]> = new Signal();
 
     private _actionMap: ActionMap;
-    private _store: ReduxStore<RootState, RootAction>;
+    private _store!: ReduxStore<RootState, RootAction>;
+    private _storage: Storage;
```

Comment:
Can this (and `_actionMap`) be `readonly`?

Reply:
Done